### ISSUE TITLE
send test messages via RabbitMQ console?

### ISSUE THREAD

**Issue by sabernar** [rochacbruno/nameko-example#1]
How would you send test messages via the RabbitMQ console? For example, if you wanted to call the compute function.  What exchange/queue/etc. would you have to use in order to get that working?

Thanks!

**Comment by rochacbruno** [comment]
@sabernar I would open a `nameko shell --broker=...` and then use the rpc client exposed there to call the compute function via service.

I don't know a way to do that direct on rabbit, because there are scenarios when you have multiple MQ nodes, also the queue ID is dynamic.

**Comment by rochacbruno** [comment]
https://github.com/rochacbruno/nameko-example#testing-it

**Comment by sabernar** [comment]
Yes, hitting the service from the shell works just fine, but I need to be able to simulate that through RabbitMQ.  I'm trying to understand how RabbitMQ gets the message to the proper nameko function.  If you took your example code and set it up with a live RabbitMQ server, how would you configure the RabbitMQ message in order to trigger your code?

All the nameko rpc examples that I see on the internet are trivial examples that work fine from the nameko shell, but if I want to use nameko in the real world, it has to be able to respond to a real, live RabbitMQ server that is queueing up messages from an external source.

**Comment by rochacbruno** [comment]
@sabernar sorry, I dont have the answer, you can ask such question on nameko-dev@googlegroups.com and point the link to this issue there, people there will know the answer.
